src/config/consts.js

```diff
@@ -2,6 +2,42 @@ const ONCE_A_DAY = 'ONCE_A_DAY';
 const ONCE_PER_HOUR = 'ONCE_PER_HOUR';
 
 export const consts = {
+  a11yLabel: {
+    address: 'Adresse',
+    backIcon: 'Zurück Taste',
```

Comment:
(Taste) needs brackets

Reply:
solved in 71c498a949d6f531126e672bc262773d65a2cbf2